recipes/data.yml

```diff
@@ -54,3 +54,10 @@ graylog-sidecar:
 graylog-sidecar-repository:
   version_major: '1.1'
   version: 1.1.0
+
+graylog-illuminate:
+  source: file:///home/donald/src/fpm-recipes/recipes/graylog-illuminate/build/graylog_illuminate.v1_5A1.gl3_3.tar.gz
```

Comment:
How about using a well-known path in the workspace here and then use a "File Parameter" for the Jenkins job? That way we can upload the tarball in the form when starting the Jenkins job.

Reply:
I didn't even know file parameter was one of the options. Looks interesting.

Shouldn't it be a standard that a release be reproducible? I think we should be building from source in the CI system itself. Not from an artifact off someone's machine. 

I will say I didn't intent for `/home/donald` to slip in there. It's a bit of a hack, since the whole line will be replaced by the pipeline.

Reply:
Ah, okay. :smile: Then let's use a placeholder value that doesn't look like a real path. :wink:

Reply:
Okay, should look better now.